client/signup/header/style.scss

```diff
@@ -2,45 +2,53 @@
 // the signup flow.
 .header {
 	position: absolute;
-		top: 0;
-		right: 0;
-		left: 0;
+	top: 0;
+	right: 0;
+	left: 0;
```

Comment:
Our code guidelines actually say we're supposed to indent these values.

Reply:
Looks like the linter or something in Github is automatically removing the extra indents; I just re-added and committed them, but they're not taking.

Reply:
Strange. Not a blocker by any means.